pkg/scraper/jobcenter_scraper.go

```diff
@@ -41,13 +70,22 @@ func ScrapeJobcenter() []model.Job {
 	// Limit to two pages
 	for i := 1; i < 3; i++ {
 
-		url := fmt.Sprintf("https://www.jobcentrebrunei.gov.bn/web/guest/search-job?q=&delta=200&start=%s", strconv.Itoa(i))
+		url := fmt.Sprintf("https://www.jobcentrebrunei.gov.bn/web/guest/search-job?q=&delta=%s&start=%s", strconv.Itoa(delta), strconv.Itoa(i))
 
-		if err := c.Visit(url); err != nil {
+		if err := linksCollector.Visit(url); err != nil {
 			fmt.Println("Error: ", err)
 			break
 		}
 	}
 
+	for i := range jobs {
+		jobs[i].Description = descriptions[i]
+	}
```

Comment:
I don't really understand what this is doing.

Is it bulk getting all descriptions and adding it to each job?

If so, is there a guarantee that the 2 slices will always be in order?